### PR TITLE
TT : TTtoXX-3Jets_madgraphMLM-pythia8

### PR DESCRIPTION
**Pull request by sihyunjeon**

### CODE REVIEW

File: Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_madgraphMLM-pythia8/TTto2L2Nu-3Jets_madgraphMLM-pythia8.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer.dat", "PartonShower/madgraphMLM-pythia8.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'JetMatching:qCut = 70.'",
+            "'JetMatching:nQmatch = 5'",
+            "'JetMatching:nJetMax = 3'",
+            "'TimeShower:mMaxGamma = 4.0'"
```
Comment (agrohsje) [inline Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_madgraphMLM-pythia8/TTto2L2Nu-3Jets_madgraphMLM-pythia8.json:20]: We leave this at 4 GeV as we are providing ttZ in 1-4 GeV and 4-10 GeV? 
20/70 are the xqcut and qcut previously used ?

**Comment by agrohsje** [comment]
What is the rule to specify L+/L- but nothing about the corresponding Nu ? 
We could still make it more consistent:
https://cms-pdmv.cern.ch/mcm/requests?page=0&prepid=GEN*&dataset_name=*Lminus*&member_of_campaign=*wmLHE*
https://cms-pdmv.cern.ch/mcm/requests?page=0&prepid=GEN*&dataset_name=*Lplus*&member_of_campaign=*wmLHE*
What do you think?

File: Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_madgraphMLM-pythia8/TTto2L2Nu-3Jets_madgraphMLM-pythia8.json (hunk 1 above)
Comment (sihyunjeon) [reply Cards/MadGraph5_aMCatNLO/TT/TTto2L2Nu-3Jets_madgraphMLM-pythia8/TTto2L2Nu-3Jets_madgraphMLM-pythia8.json:20]: Ah sorry I forgot about this myself. We should use 1.0GeV this time and then move up to 4.0 from 2023 campaign

**Comment by sihyunjeon** [comment]
> What is the rule to specify L+/L- but nothing about the corresponding Nu ? We could still make it more consistent: https://cms-pdmv.cern.ch/mcm/requests?page=0&prepid=GEN*&dataset_name=*Lminus*&member_of_campaign=*wmLHE* https://cms-pdmv.cern.ch/mcm/requests?page=0&prepid=GEN*&dataset_name=*Lplus*&member_of_campaign=*wmLHE* What do you think?

We discussed this before and our conclusion was "nu doesn't matter since we don't really detect them"

**Comment by agrohsje** [comment]
Did we? Seems I forgot. But does it really make sense? We can only distinguish b from bbar with 60% probability. Not so different from 50:50. Wouldn't it be cleaner to distinguish particles from antiparticles wherever we deal with split cases ?

**Comment by agrohsje** [comment]
There is some remaining charge flip from oscillations that we will never escape/improve. But my comment is more about avoiding too many exceptions in the name. I think the Lplus Lminus is rare so we could be complete for the nus.

**Comment by sihyunjeon** [comment]
> There is some remaining charge flip from oscillations that we will never escape/improve. 
This I disagree with the same comment above. We don't know if the jet charge improvement will ultimately lead us to 65% or 85% at the end of Run3 but still this is something that people will be willing to make improvements. Same for charm quarks, we don't have charm quark samples in our list but although identification prob is low, i think it's worth specifying this with not so good identification prob for now.

> But my comment is more about avoiding too many exceptions in the name.
This I agree, depends on how much and how long we concentrate on this. Let's think about this.

**Comment by agrohsje** [comment]
What about maxgamma here? Did we change our mind or should this still be reduced to fit powheg hvq @sihyunjeon?

**Pull request merged**